Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,86 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## Notify listener
+
+By call this method whenever the datasource changes, to notify any clients the datasource may have. We have provided support to perform three operations like Add, Remove and Reset. NotifyListener is used to notify the dataSource may have changed.
+
+#### Add Action
+
+We need to notify the datasource changes due to add the appointment from the collection. 
+
+{% tabs %}
+{% highlight dart hl_lines="14 15 16" %}
+
+@override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+          floatingActionButton: FloatingActionButton(
+            child: const Text('Add appointment'),
+            onPressed: () {
+              final Appointment app = Appointment(
+                  startTime: _controller.displayDate!,
+                  endTime:
+                      _controller.displayDate!.add(const Duration(hours: 2)),
+                  subject: 'Add Appointment',
+                  color: Colors.pink);
+              _events?.appointments!.add(app);
+              _events?.notifyListeners(
+                  CalendarDataSourceAction.add, <Appointment>[app]);
+            },
+          ),
+          body: SfCalendar(
+            view: CalendarView.month,
+            controller: _controller,
+            dataSource: _events,
+          )),
+    );
+  }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Remove Action
+We need to notify the datasource changes due to remove the appointment from the collection.
+
+{% tabs %}
+{% highlight dart hl_lines="2 3 4" %}
+
+final Appointment removeAppointment = appointmentCollection[0];
+appointmentCollection.remove(removeAppointment);
+_events?.notifyListeners(CalendarDataSourceAction.remove,
+    <Appointment>[removeAppointment]);
+
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Reset Action
+We need to notify the datasource changes due to reset the appointments.
+
+{% tabs %}
+{% highlight dart hl_lines="1 2 3" %}
+
+_events?.appointments!.clear();
+_events?.notifyListeners(
+    CalendarDataSourceAction.reset, _events!.appointments!);
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Add Resource
+
+We need to notify the datasource changes due to add the resource from the collection.
```

Comment:
Check and add sample for Add/remove/reset resource also

Reply:
I have checked and added the sample for the Add resource, remove resource and reset resource. I have updated the changes in my latest commit.